glass/src/libnt/native/cpp/NTField2D.cpp

```diff
@@ -198,7 +198,7 @@ void NTField2DModel::ForEachFieldObject(
         func) {
   for (auto&& obj : m_objects) {
     if (obj->Exists()) {
-      func(*obj, wpi::drop_front(obj->GetName(), m_path.size()));
+      func(*obj, wpi::remove_prefix(obj->GetName(), m_path).value());
```

Comment:
```suggestion
      if (auto name = wpi::remove_prefix(obj->GetName(), m_path)) {
        func(*obj, *name);
      }
```
Avoids the potential exception.